### ISSUE TITLE
bw cli 1.17 throws npm err about 'Unsupported engine' and then missing 'tldjs'

### ISSUE THREAD

**Issue by luckman212** [bitwarden/cli#343]
## Versions

```
$ sw_vers
ProductName:	macOS
ProductVersion:	11.4
BuildVersion:	20F71

$ node -v
v16.4.0

$ npm -v
7.19.0
```

## Describe the Bug

```
$ npm -g i @bitwarden/cli
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@bitwarden/cli@1.17.0',
npm WARN EBADENGINE   required: { node: '~14', npm: '~7' },
npm WARN EBADENGINE   current: { node: 'v16.4.0', npm: '7.19.0' }
npm WARN EBADENGINE }

$ bw
node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module 'tldjs'
Require stack:
- /Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js:140:18)
    at __webpack_require__ (/Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js:21:30)
    at Module.<anonymous> (/Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js:907:23)
    at __webpack_require__ (/Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js:21:30)
    at /Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js:85:18
    at Object.<anonymous> (/Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js:88:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/luke/.n/lib/node_modules/@bitwarden/cli/build/bw.js' ]
}
```

**Comment by latipun7** [comment]
I have an issue too. Run `bw --help` and all commands error.
```bash
❯ bw --help
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'tldjs'
Require stack:
- /home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:140:18)
    at __webpack_require__ (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:21:30)
    at Module.<anonymous> (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:907:23)
    at __webpack_require__ (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:21:30)
    at /home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:85:18
    at Object.<anonymous> (/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js:88:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib/node_modules/@bitwarden/cli/build/bw.js'
  ]
}
```

OS: Linux WSL (`Linux DESKTOP-GVN6OUJ 4.19.104-microsoft-standard #1 SMP Wed Feb 19 06:37:35 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`)

Node: 14.17.1

Bitwarden/cli via npm:
```log
❯ npm -g list
/home/latipun7/.fnm/node-versions/v14.17.1/installation/lib
├── @bitwarden/cli@1.17.0
├── npm@7.19.0
└── yarn@1.22.10
```

related: #342

**Comment by eliykat** [comment]
The `unsupported engine` error is because we've specified node v14 as being the recommended/compatible node runtime. The warning is advisory only - you're free to use a different node version, and I am not actually aware of any incompatibilities. However, it makes support and troubleshooting easier if we have officially decided on a supported node version.

The `tldjs` error is unrelated to the node version. I've been able to reproduce it from the npm package. However, it is a duplicate of #342, so I'll close this issue and tackle it over there instead.
